### PR TITLE
[WIP] fix: (core) adding visual representation of label to the busy indicator component.

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.
Closes :https://github.com/SAP/fundamental-ngx/issues/3111

#### Please provide a brief summary of this pull request.
adding visual representation of label to the busy indicator component. added missing example.

Screenshot 2021-05-04 at 9 57 06 PM

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/busy-indicator/busy-indicator.module.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { BusyIndicatorComponent } from './busy-indicator.component';
+import { FormModule } from '../form/form.module';
 
 @NgModule({
     declarations: [BusyIndicatorComponent],
     exports: [BusyIndicatorComponent],
-    imports: [CommonModule]
+    imports: [CommonModule, FormModule]
```
Comment (dimamarksman) [inline libs/core/src/lib/busy-indicator/busy-indicator.module.ts:9]: Do we really need "FormModule"?

**Review by kavya-b — COMMENTED** [review]
This looks like the `Extended Indicator` in the Fiori visual specs and should probably be implemented in Fundamental Styles first because there are specific values for spacings and width, etc. 
image

**Comment by Lokanathan-k** [comment]
> This looks like the `Extended Indicator` in the Fiori visual specs and should probably be implemented in Fundamental Styles first because there are specific values for spacings and width, etc.
> image

@kavya There are lot of cleanup required for this component, we take it up in a separate PR.

**Review by platon-rov — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
> > This looks like the `Extended Indicator` in the Fiori visual specs and should probably be implemented in Fundamental Styles first because there are specific values for spacings and width, etc.
> > image
> 
> @kavya There are lot of cleanup required for this component, we take it up in a separate PR.

I agree with @kavya-b

**Pull request closed without merging**